From 71e6051e75ad712728fd291a3e787c4702d64bff Mon Sep 17 00:00:00 2001
From: Ofer Heifetz <oferh@marvell.com>
Date: Mon, 11 Nov 2019 15:19:27 +0200
Subject: [PATCH 1002/1239] arm64: dts: a8k: add spi direct access mapping

In ATF, spi direct mapping window of [0xf9000000, 0xf9ffffff] is added
for a80x0_ocp, this patch adds the spi direct access mapping for
corresponding board in u-boot.

Change-Id: I94441bb73558e0d3392b80d12d48e62623b2c908
Signed-off-by: Ofer Heifetz <oferh@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/18470
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Kostya Porotchkin <kostap@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/18951
Tested-by: Kostya Porotchkin <kostap@marvell.com>
Reviewed-by: Marcin Wojtas <marcin@marvell.com>
---
 arch/arm/dts/armada-8040-ocp.dts | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/dts/armada-8040-ocp.dts b/arch/arm/dts/armada-8040-ocp.dts
index 460f2d49bf..b2af482496 100644
--- a/arch/arm/dts/armada-8040-ocp.dts
+++ b/arch/arm/dts/armada-8040-ocp.dts
@@ -241,6 +241,9 @@
 &cp1_spi1 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&cp1_spi1_pins>;
+	reg = <0x700680 0x50>,		/* control */
+		<0x2000000 0x1000000>,	/* CS0 */
+		<0 0xffffffff>;		/* CS1 */
 	status = "okay";
 
 	spi-flash@0 {
-- 
2.29.0

